modules/admin_manual/pages/configuration/user/oidc/index.adoc

```diff
@@ -3,56 +3,216 @@
 :toclevel: 2
 :openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
 :openid-connect-url: https://openid.net/connect/
+:openid-app-marketplace-url: https://marketplace.owncloud.com/apps/openidconnect
+:openid-app-github-url: https://github.com/owncloud/openidconnect
+:mod_rewrite-docs-url: https://httpd.apache.org/docs/current/mod/mod_rewrite.html
+:install-libsodium-url: https://www.php.net/manual/en/sodium.installation.php 
```

Comment:
no `lib-sodium`. it is already complied in php...

Reply:
From PHP 7.2, yes.